### PR TITLE
Add Clear button to TimeSeriesWidget

### PR DESCRIPTION
**Pull request by padawannn**
- [x] If show controls are disabled in Timeseries widget you cannot clear the Timeseries filter --> add Clear btn, as in the rest of widgets
- [x] Also enable the speed button, even though the animation has not started

### CODE REVIEW

**Review by Clebal — APPROVED** [review]
LGTM

File: packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js [hunk 1]
```diff
@@ -233,4 +233,18 @@ describe('TimeSeriesWidgetUI', () => {
       jest.runAllTimers();
     });
   });
+
+  test('clear when animation is working', () => {
+    const onStop = jest.fn();
+    render(<Widget isPlaying={true} onStop={onStop} />);
+    fireEvent.click(screen.getByText(/Clear/));
+    setTimeout(() => expect(onStop).toHaveBeenCalledTimes(1), 250);
```
Comment (VictorVelarde) [inline packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js:241]: Can't we avoid using a setTimeout for expect? Have you tried with async() and then screen.findByText(/Clear/)

Comment (VictorVelarde) [reply packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js:241]: Inside testingjavascript materials, there is a little utility, called fire-event-async, that waits for next tick.
That could be useful here:

image

PD: video from https://testingjavascript.com/lessons/svelte-use-dom-testing-library-with-svelte (01:26)

Comment (VictorVelarde) [reply packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js:241]: So we can try to get a code with something like:

```
    const onStop = jest.fn();
    render(<Widget timeWindow={[1517785200000, 1528063200000]} onStop={onStop} />);
    fireEventAsync.click(await screen.findByText(/Clear/));
    expect(onStop).toHaveBeenCalledTimes(1);
```

**Review by VictorVelarde — COMMENTED** [review]
Small comment on tests to check, but LGTM

File: packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js (hunk 1 above)
Comment (padawannn) [reply packages/react-ui/__tests__/widgets/TimeSeriesWidgetUI.test.js]: @VictorVelarde I've removed the `setTimeout `

**Pull request merged**